spec/unit/puppet-lint/checkplugin_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'spec_helper'
+
+class DummyCheckPlugin < PuppetLint::CheckPlugin
+  def check
+    # Since we're calling `tokens` from a `check` method, we should get our own Array object.
+    # If we add an extra token to it, PuppetLint::Data.tokens should remain unchanged.
+    tokens << :extra_token
+  end
+
+  def fix
+    tokens << :fix_token
```

Comment:
I don't think this is supposed to be a valid API because there's also the `add_token` to actually modify it.